packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -195,64 +207,75 @@ export const Account = ({
   const secondaryEmails = emails!.filter((email) => !email.isPrimary);
 
   return (
-    <section className="account" data-testid="account-section">
+    <section className="mt-8" data-testid="account-section">
       <ul>
-        <li className="flex justify-space-between">
-          <h3 data-testid="email-label">
+        <li className={`${styleClasses.li} flex justify-between`}>
+          <h3 data-testid="email-label" className="mt-0 my-0 mb-1 text-lg">
             <span
-              className={`${query === primaryEmail.email ? 'highlight' : ''}`}
+              className={`${
+                query === primaryEmail.email ? 'bg-yellow-100' : ''
+              }`}
             >
               {primaryEmail.email}
             </span>
           </h3>
           <span
             data-testid="verified-status"
             className={`${
-              primaryEmail.isVerified ? 'verified' : 'not-verified'
+              primaryEmail.isVerified
+                ? styleClasses.enabledOrVerified
+                : styleClasses.notEnabledOrVerified
             }`}
           >
             {primaryEmail.isVerified ? 'verified' : 'not verified'}
           </span>
         </li>
-        <li className="flex justify-space-between">
+        <li className={`${styleClasses.li}`}>
```

Comment:
Tiny nit: instead of ```={`${styleClasses.li}`}``` (and similarly on other lines like L225 - L227, L235, and others), they can be `={styleClasses.li}`.